keyboards/era/sirind/tomak/info.json

```diff
@@ -0,0 +1,710 @@
+            {"matrix": [9, 1], "x": 109, "y": 38, "flags": 4},
+            {"matrix": [9, 2], "x": 121, "y": 38, "flags": 4},
+            {"matrix": [9, 3], "x": 133, "y": 38, "flags": 4},
+            {"matrix": [9, 4], "x": 145, "y": 38, "flags": 4},
+            {"matrix": [9, 5], "x": 157, "y": 38, "flags": 4},
+            {"matrix": [9, 7], "x": 177, "y": 38, "flags": 1},
+            {"matrix": [10, 9], "x": 212, "y": 51, "flags": 4},
+            {"matrix": [10, 7], "x": 184, "y": 45, "flags": 1},
+            {"matrix": [10, 6], "x": 174, "y": 51, "flags": 1},
+            {"matrix": [10, 5], "x": 151, "y": 51, "flags": 4},
+            {"matrix": [10, 4], "x": 139, "y": 51, "flags": 4},
+            {"matrix": [10, 3], "x": 127, "y": 51, "flags": 4},
+            {"matrix": [10, 2], "x": 115, "y": 51, "flags": 4},
+            {"matrix": [10, 1], "x": 102, "y": 51, "flags": 4},
+            {"matrix": [10, 0], "x": 90, "y": 51, "flags": 4},
+            {"matrix": [11, 1], "x": 96, "y": 64, "flags": 1},
+            {"matrix": [11, 2], "x": 110, "y": 64, "flags": 4},
+            {"matrix": [11, 4], "x": 137, "y": 64, "flags": 1},
+            {"matrix": [11, 5], "x": 153, "y": 64, "flags": 1},
+            {"matrix": [11, 6], "x": 168, "y": 64, "flags": 1},
+            {"matrix": [11, 7], "x": 183, "y": 64, "flags": 1},
+            {"matrix": [11, 8], "x": 200, "y": 64, "flags": 1},
+            {"matrix": [11, 9], "x": 212, "y": 64, "flags": 1},
+            {"matrix": [11, 10], "x": 224, "y": 64, "flags": 1},
+            {"x": 224, "y": 40, "flags": 8},
+            {"x": 224, "y": 45, "flags": 8},
+            {"x": 224, "y": 50, "flags": 8}
+        ],
+        "max_brightness": 120,
+        "sleep": true,
+        "split_count": [40, 59]
+    },
+    "split": {
+        "bootmagic": {
+            "matrix": [6, 0]
+        },
+        "enabled": true,
+        "matrix_pins": {
+            "right": {
+                "cols": ["GP2", "GP3", "GP4", "GP5", "GP6", "GP7", "GP8", "GP9", "GP27", "GP17", "GP13"],
+                "rows": ["GP10", "GP11", "GP12", "GP28", "GP19", "GP16"]
+            }
+        },
+        "transport": {
+            "protocol": "serial_usart",
+            "sync": {
+                "indicators": true,
+                "layer_state": true,
+                "matrix_state": true
+            }
+        }
+    },
+    "url": "",
+    "usb": {
+        "device_version": "1.0.2",
+        "pid": "0x0006",
+        "vid": "0x4552"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP20"
+    },
+    "layout_aliases": {
+        "LAYOUT_all": "LAYOUT_split_tkl_ansi_split_rshift_bs_rspace",
+        "LAYOUT_tomak_ansi": "LAYOUT_split_tkl_ansi",
+        "LAYOUT_tomak_ansi_split_bs": "LAYOUT_split_tkl_ansi_split_bs",
+        "LAYOUT_tomak_ansi_split_rshift": "LAYOUT_split_tkl_ansi_split_rshift",
+        "LAYOUT_tomak_ansi_split_rshift_bs": "LAYOUT_split_tkl_ansi_split_rshift_bs"
```

Comment:
Layout names should not contain the keyboard name in them.

Reply:
I edited it. :)